src/libs/elektra/kdb.c

```diff
@@ -677,7 +677,10 @@ static int elektraGetCheckUpdateNeeded (Split * split, Key * parentKey)
 
 			backendUpdateSize (backend, split->parents[i], 0);
 		}
-		// TODO: set error in else case!
+		else
+		{
```

Comment:
Please also call ELEKTRA_SET_ERROR with some information that there is no resolver plugin or kdbGet was missing in the resolver plugin.